backend/de.metas.material/dispo-commons/src/main/java/de/metas/material/dispo/commons/repository/CandidateRepositoryWriteService.java

```diff
@@ -344,6 +349,9 @@ void updateCandidateRecordFromCandidate(
 			@NonNull final Candidate candidate,
 			final boolean preserveExistingSeqNo)
 	{
+		final DimensionService dimensionService = SpringContextHolder.instance.getBean(DimensionService.class);
```

Comment:
same thing: since CandidateRepositoryWriteService is also a spring service, i think you can inject DimensionService via constructor

Reply:
done